src/ui/Views/Course/Variants.cshtml

```diff
@@ -1,94 +1,174 @@
+              </dd>
+              <dt class="course-parts__fields__label">
+                Fee details (optional)
+              </dt>
+              <dd class="course-parts__fields__value">
+                This field is empty
+              </dd>
+              <dt class="course-parts__fields__label">
+                Financial support you offer (optional)
+              </dt>
+              <dd class="course-parts__fields__value">
+                This field is empty
+              </dd>
+            </dl>
+          </div>
+          <div class="course-parts__item">
+            <h3 class="course-parts__title">Requirements and eligbility</h3>
+            <a class="course-parts__action" href="@Url.Action("Eligbility", "Course", new { ucasCode = Model.Course.UcasCode })">Edit</a>
+            <dl class="course-parts__fields">
+              <dt class="course-parts__fields__label">
+                Qualifications needed
+              </dt>
+              <dd class="course-parts__fields__value">
+                This field is empty
+              </dd>
+              <dt class="course-parts__fields__label">
+                Personal qualities (optional)
+              </dt>
+              <dd class="course-parts__fields__value">
+                This field is empty
+              </dd>
+              <dt class="course-parts__fields__label">
+                Other requirements (optional)
+              </dt>
+              <dd class="course-parts__fields__value">
+                This field is empty
+              </dd>
+            </dl>
+          </div>
+        </div>
+      </div>
+   </div>
+
+   <h3 class="govuk-heading-m">Training locations</h3>
+   <table class="govuk-table">
+      <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+          <th class="govuk-table__header" scope="col">Location name</th>
+          <th class="govuk-table__header" scope="col">Address</th>
+          <th class="govuk-table__header" scope="col">Code</th>
+          <th class="govuk-table__header" scope="col">Applications accepted from</th>
+        </tr>
         </thead>
         <tbody class="govuk-table__body">
-            @foreach (var school in Model.Course.Schools)
-            {
-                <tr class="govuk-table__row">
-                    <td class="govuk-table__cell">@school.LocationName</td>
-                    <td class="govuk-table__cell">@school.Address</td>
-                    <td class="govuk-table__cell">@school.Code</td>
-                    @*<td class="govuk-table__cell">Vacancies</td>
-                        <td class="govuk-table__cell">Not applicable</td>*@
-                    <td class="govuk-table__cell">@school.ApplicationsAcceptedFrom</td>
-                </tr>
-            }
-        </tbody>
-    </table>
-
-    <p>
-        <a href="@link" class="govuk-link">Contact the Becoming a Teacher team to report a problem</a>
-    </p>
+          @foreach (var school in Model.Course.Schools)
```

Comment:
See new design for training locations here:
https://manage-courses-prototype.herokuapp.com/history/course-not-running
https://manage-courses-prototype.herokuapp.com/course/the-university-of-reading/36W4